packages/design-system/src/components/slider/slider.stories.tsx

```diff
@@ -0,0 +1,490 @@
+  },
+  includeTextField: {
+    control: {
+      type: 'boolean',
+    },
+    defaultValue: true,
+  },
+  includeRangeLabel: {
+    control: {
+      type: 'boolean',
+    },
+    defaultValue: true,
+  },
+  includeOutputField: {
+    control: {
+      type: 'boolean',
+    },
+    defaultValue: false,
+  },
+  minValue: {
+    control: {
+      type: 'number',
+    },
+    defaultValue: 0,
+  },
+  maxValue: {
+    control: {
+      type: 'number',
+    },
+    defaultValue: 100,
+  },
+};
+
+export const SliderExample: Story<SliderProps> = ({
+  label,
+  minValue,
+  maxValue,
+  includeRangeLabel,
+  includeTextField,
+  includeOutputField,
+  ...rest
+}) => {
+  const [open, setOpen] = useState<boolean>(false);
+
+  return (
+    <Slider {...rest}>
+      {({ state }) => (
+        <>
+          {includeRangeLabel && <AriaLabel>{label}</AriaLabel>}
+          {includeOutputField && <AriaText>{state.values.join(',')}</AriaText>}
+          {includeTextField && (
+            <NumberField
+              value={state.values[0]}
+              onChange={(v) => state.setThumbValue(0, v)}
+            >
+              <Input />
+            </NumberField>
+          )}
+          <SliderTrack>
+            {({ state }: SliderRenderProps) => (
+              <>
+                <div
+                  className={sliderBar}
+                  style={{
+                    position: 'absolute',
+                    left: state.getThumbValue(1)
+                      ? `${state.getThumbValue(0)}%`
+                      : 0,
+                    width: state.getThumbValue(1)
+                      ? `${(state.getThumbValue(1) || 0) - (state.getThumbValue(0) || 0)}%`
+                      : `${state.getThumbValue(0)}%`,
+                  }}
+                />
+                <TooltipTrigger isOpen={open}>
+                  <SliderThumb
+                    onHoverChange={(isHovered) => setOpen(isHovered)}
+                  />
+                  <Tooltip>{state.values[0]}</Tooltip>
+                </TooltipTrigger>
+              </>
+            )}
+          </SliderTrack>
+          <AriaText slot='min'>{minValue || 0}</AriaText>
+          <AriaText slot='max'>{maxValue || 100}</AriaText>
+        </>
+      )}
+    </Slider>
+  );
+};
+
+SliderExample.storyName = 'Slider with input';
+
+SliderExample.argTypes = {
+  ...args,
+};
+
+export const SliderExampleWithOutput: Story<SliderProps> = ({
+  label,
+  minValue,
+  maxValue,
+  includeRangeLabel,
+  includeTextField,
+  includeOutputField,
+  ...rest
+}) => {
+  const [open, setOpen] = useState<boolean>(false);
+
+  return (
+    <Slider {...rest}>
+      {({ state }) => (
+        <>
+          {includeRangeLabel && <AriaLabel>{label}</AriaLabel>}
+          {includeOutputField && <AriaText>{state.values.join(',')}</AriaText>}
+          <SliderTrack>
+            {({ state }: SliderRenderProps) => (
+              <>
+                <div
+                  className={sliderBar}
```

Comment:
i think i applied this incorrectly, but was unable to get it working with the same method the `Picker` stories were using. User error, i'm sure 😆

Reply:
removed and added to the regular `slider.css.ts` file since this div was turned into a component